test/e2e/framework.go

```diff
@@ -1529,7 +1536,243 @@ func (data *TestData) Cleanup(namespaces []string) {
+// 		files = strings.Split(stdout, "\n")
+// 		fmt.Printf("found meta file at: %v with stdout: %v\n", files, stdout)
+// 	}
+// 	cmd = "/bin/sh -c find / -name 'covcounters.*' -exec basename {} ';'"
+// 	rc, stdout, stderr, err = data.RunCommandOnNode(nodeName, cmd)
+// 	if err != nil || rc != 0 {
+// 		if err != nil && !strings.Contains(err.Error(), "No such file or directory") {
+// 			return fmt.Errorf("error when running this find command (for coverage file) '%s' on Node %s, Pod %s, stderr: <%v>, err: <%v>", cmd, nodeName, podName, stderr, err)
+// 		}
+// 	} else {
+// 		stdout = strings.TrimSpace(stdout)
+// 		files = append(files, strings.Split(stdout, "\n")...)
+// 		fmt.Printf("found cover file at: %v with stdout: %v\n", strings.Split(stdout, "\n"), stdout)
+// 	}
+// 	for _, file := range files {
+// 		if len(file) == 0 {
+// 			continue
+// 		}
+// 		err = data.copyNodeFiles(nodeName, file, covDir, covPrefix)
+// 		if err != nil {
+// 			return fmt.Errorf("error when copying coverage files from Pod '%s' to coverage directory '%s': %v", podName, covDir, err)
+// 		}
+// 	}
+// 	return nil
+// }
+
+// func (data *TestData) killProcessesAndCollectCovFiles(namespace, podName, containerName, processName, covDir, covPrefix string) error {
+// 	cmds := []string{"pgrep", "-f", processName}
+// 	stdout, stderr, err := data.RunCommandFromPod(namespace, podName, containerName, cmds)
+// 	if err != nil {
+// 		return fmt.Errorf("error when getting pid of '%s', stderr: <%v>, err: <%v>", processName, stderr, err)
+// 	}
+// 	cmds = []string{"kill", "-SIGINT", strings.TrimSpace(stdout)}
+// 	log.Infof("Sending SIGINT to '%s' with cmd 'kill -SIGINT %s'", processName, strings.TrimSpace(stdout))
+// 	fmt.Println("sending SIGINT")
+// 	_, stderr, err = data.RunCommandFromPod(namespace, podName, containerName, cmds)
+// 	if err != nil {
+// 		return fmt.Errorf("error when sending SIGINT signal to '%s', stderr: <%v>, err: <%v>", processName, stderr, err)
+// 	}
+// 	log.Infof("Copying coverage files from worker nodes kind-worker and kind-worker2")
+// 	if err = data.findAndCopyCovFiles("kind-worker", podName, covDir, covPrefix); err != nil {
+// 		fmt.Println("errored out looking in kind-worker")
+// 		return err
+// 	}
+// 	fmt.Println("did not error out looking in kind-worker")
+// 	if err = data.findAndCopyCovFiles("kind-worker2", podName, covDir, covPrefix); err != nil {
+// 		fmt.Println("errored out looking in kind-worker2")
+// 		return err
+// 	}
+// 	fmt.Println("did not error out looking in kind-worker2")
+// 	return nil
+// }
+
+// docker cp kind-worker:/var/log/cm-coverage/. .coverage/clickhouse-monitor-coverage/
+func (data *TestData) copyCovFolder(nodeName, covDir, covPrefix string) error {
+	covDirAbs, err := filepath.Abs("../../" + covDir)
+	if err != nil {
+		return fmt.Errorf("copyCovFolder: error creating absolute file path: %v", err)
+	}
+	cmd := exec.Command("docker", "cp", nodeName+":"+"/var/log/"+covPrefix+"-coverage/.", covDirAbs);
```

Comment:
make a separate string beforehand, instead of using "+" operator in this line  
```suggestion
	cpCmd := nodeName+":"+"/var/log/"+covPrefix+"-coverage/."
	cmd := exec.Command("docker", "cp", cpCmd, covDirAbs);
```